docs/UsersGuide/source/ConfigNewPlatform.rst

```diff
@@ -101,7 +101,13 @@ Prior to building the UFS SRW Application on a new machine, you will need to ins
    make -j4 2>&1 | tee log.make
    make deploy 2>&1 | tee log.deploy
 
-As with NCEPLIBS-external, the above commands go through the process of cloning the git repository for NCEPLIBS, creating and entering a build directory, and invoking cmake and make to build the code. The ``make deploy`` step created a number of modulefiles and scripts that will be used for setting up the build environment for the UFS SRW App. The ``ESMFMKFILE`` variable allows NCEPLIBS to find the location where ESMF has been built.
+As with NCEPLIBS-external, the above commands go through the process of cloning the git repository for NCEPLIBS, creating and entering a build directory, and invoking cmake and make to build the code. The ``make deploy`` step created a number of modulefiles and scripts that will be used for setting up the build environment for the UFS SRW App. The ``ESMFMKFILE`` variable allows NCEPLIBS to find the location where ESMF has been built; if you receive a ``ESMF not found, abort`` error, you may need to specify a slightly different location:
+
+.. code-block:: console
+
+   export ESMFMKFILE=${INSTALL_PREFIX}/lib64/esmf.mk
```

Comment:
Note that `lib64` is specific to certain Linux distributions (Redhat, CentOS). On macOS, Ubuntu etc. this is just called `lib`.

Reply:
@mkavulich This needs to be fixed before the PR can be merged, please.

Reply:
I don't think a fix is needed, the version called "lib" is used in the earlier code block; this is explicitly for users who run into trouble with that syntax. Unless you think the current way it is organized is too unclear.

Reply:
My bad, the github diffs only show you fractions of the files. All clear and all good.